lib/board.rb

```diff
@@ -0,0 +1,81 @@
+class Board
+  attr_reader :grid
+
+  def initialize(grid = [[1, 2, 3], [4, 5, 6], [7, 8, 9]])
+    @grid = grid
+  end
+
+  def get_value(position)
+    line, column = get_line_column(position)
+    @grid[line][column]
```

Comment:
Coupling: Accessing the `@grid` instance var directly, even within the same class, ties this code more tightly to an implementation.  (Currently the implementation of `@grid` is as an instance variable.  If the implementation of `@grid` became something else (eg a method call), this line would need to change.)